dev/breeze/doc/09_release_management_tasks.rst

```diff
@@ -189,6 +189,26 @@ These are all of the available flags for the ``release-prod-images`` command:
   :width: 100%
   :alt: Breeze release management release prod images
 
+Adding git tags for providers
+"""""""""""""""""""""""""""
```

Comment:
There are a few quotes missing. You need to cover the heading with quotes

Reply:
Ack